Sources/PortalSwift/Swaps/PortalSwaps.swift

```diff
@@ -111,10 +109,20 @@ public class PortalSwaps {
   private var apiKey: String
   private var portal: Portal
 
-  init(apiKey: String, portal: Portal) {
+  public init(apiKey: String, portal: Portal) {
     self.apiKey = apiKey
     self.portal = portal
   }
+  
+  public func getQuote(args: QuoteArgs, forChainId: String, completion: @escaping (Result<Quote>) -> Void) {
+    do {
+      try self.portal.api.getQuote(self.apiKey, args, forChainId) { result in
+        completion(result)
+      }
+    } catch {
+      completion(Result(error: error))
+    }
+  }
 
   public func getQuote(args: QuoteArgs, completion: @escaping (Result<Quote>) -> Void) {
```

Comment:
will these functions still work or get used with the new api?

Reply:
Yeah, both will work, I made `getQuote` overloaded so that there's 1 with the `forChainId` argument so customers can specify exactly what chain they want to use and another that excludes it (for backwards compatibility).